src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -109,6 +111,34 @@ func runClientProxy(cfg *config.Config) {
 	// Configure client initially
 	client.Configure(cfg.Client)
 
+	// this flag specifies the port to open the HTTP server
+	// between the UI and
+	// with a corresponding HTTP server at
+	// the following address
+	if cfg.Client.UiAddr != "" {
+		wlChan := make(chan *whitelist.Config)
+		go func() {
+			http.ListenAndServe(cfg.Client, configUpdates, wlChan)
```

Comment:
It would be useful to catch the error that ListenAndServe returns, such as when the port is already used by another process. Otherwise we wouldn't know why the server didn't start.

If we depend on this server to continue, we may as well mark this error as fatal.